frontend/src/employee-frontend/components/messages/Sidebar.tsx

```diff
@@ -81,9 +81,16 @@ interface AccountsParams {
   setSelectedReceivers: React.Dispatch<
     React.SetStateAction<SelectorNode | undefined>
   >
+  selectedUnit: SelectOptionProps | undefined
+  setSelectedUnit: (unit: SelectOptionProps) => void
 }
 
-function Accounts({ accounts, setSelectedReceivers }: AccountsParams) {
+function Accounts({
+  accounts,
+  setSelectedReceivers,
+  selectedUnit,
+  setSelectedUnit
```

Comment:
These could be taken straight from the context instead of passing them through the whole component tree.